app/collections/playlists.js

```diff
@@ -1,9 +1,47 @@
 import Backbone from 'backbone';
 import Playlist from '../models/playlist';
+import cozysdk from 'cozysdk-client';
 
 
 const Playlists = Backbone.Collection.extend({
-    model: Playlist
+   	
+   	model: Playlist,
+
+    initialize: function() {
+        this.on('add', this.onAdd, this);
+    },
+
+    onAdd: function(playlist) {
+        if (!playlist.get('_id')) {
+            playlist.save()
+        }
+    },
+    
+    comparator: function (collection) {
+        return collection.get('title');
+    },
+
+    sync: function (method, model, options) {
+        if (method == 'read') {
+            console.log('fetch playlist');
```

Comment:
Please avoid `console` calls from code, except if it's needed for app feedback (like `errors`). Else, it'll will unnecessary bloat the code.